libs/src/services/ethWeb3Manager/index.js

```diff
@@ -29,11 +29,15 @@ class EthWeb3Manager {
 
   async sendTransaction (
     contractMethod,
-    gasAmount = DEFAULT_GAS_AMOUNT,
     contractAddress = null,
     privateKey = null,
-    txRetries = 5
+    txRetries = 5,
+    txGasLimit = null
   ) {
+    const gasLimit = txGasLimit || await estimateGas({
```

Comment:
yep! that was the thinking. who knows how accurate this will be in the long run and there maybe some complicated tx actions that we do need to build a bigger buffer into